### PR TITLE
Add buildifier and skylint checks

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,11 @@
     "deploy": "gulp deploy:devapp",
     "webdriver-manager": "webdriver-manager",
     "docs": "gulp docs",
-    "api": "gulp api-docs"
+    "api": "gulp api-docs",
+    "preskylint": "bazel build --noshow_progress @io_bazel//src/tools/skylark/java/com/google/devtools/skylark/skylint:Skylint",
+    "skylint": "find . -type f -name \"*.bzl\" ! -path \"*/node_modules/*\" ! -path \"./dist/*\" | xargs $(bazel info bazel-bin)/external/io_bazel/src/tools/skylark/java/com/google/devtools/skylark/skylint/Skylint",
+    "prebuildifier": "bazel build --noshow_progress @com_github_bazelbuild_buildtools//buildifier",
+    "buildifier": "find . -type f \\( -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs $(bazel info bazel-bin)/external/com_github_bazelbuild_buildtools/buildifier/*/buildifier"
```
Comment (jelbourn) [inline package.json]: This won't work for our contributors on Windows

File: package.json [hunk 2]
```diff
@@ -18,7 +18,11 @@
     "deploy": "gulp deploy:devapp",
     "webdriver-manager": "webdriver-manager",
     "docs": "gulp docs",
-    "api": "gulp api-docs"
+    "api": "gulp api-docs",
+    "preskylint": "bazel build --noshow_progress @io_bazel//src/tools/skylark/java/com/google/devtools/skylark/skylint:Skylint",
```
Comment (jelbourn) [inline package.json]: What's with the "pre" tasks?

**Comment by jelbourn** [comment]
@alexeagle is the Windows stuff still in a state where buildifier not is the least of the problems?

**Comment by alexeagle** [comment]
No Windows generally works now. Still waiting on Bazel 0.12 to fix something we need for the karma rules under puppeteer

**Pull request closed without merging**